apps/pipelines/nodes/nodes.py

```diff
@@ -195,10 +188,7 @@ def validate_llm_model(self):
 
         # Validate model parameters
         if params_cls := LLM_MODEL_PARAMETERS.get(model.name):
-            params_cls.model_validate(
-                self.llm_model_parameters,
-                context={"temperature": self.llm_temperature},
```

Comment:
Since the temperature is now available as model params, we don't have to pass it in as context